examples/grid.cpp

```diff
@@ -54,9 +55,31 @@ extern "C" {
 #include "common/common_def.h"
 #include "common/condition.h"
 #include "common/lock.h"
+#include "font.h"
 
 using namespace std;
 
+class Timer
+{
+public:
+    Timer(const char* str):m_str(str)
```

Comment:
In C++, probably best to avoid raw pointers when possible.

    Timer(const std::string& str)